examples/flutter_view/ios/Runner/MainViewController.m

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.Copyright © 2017 The Chromium Authors. All rights reserved.
+
+
+#import <Foundation/Foundation.h>
+
+#import "MainViewController.h"
+#import "NativeViewController.h"
+
+@interface MainViewController ()
+
+@property(strong, nonatomic) NSString* messageName;
+@property (strong, nonatomic) NativeViewController* nativeViewController;
+@property (strong, nonatomic) FlutterViewController* flutterViewController;
+@end
+
+static NSString* const emptyString = @"";
+
+@implementation MainViewController
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+}
+
+- (void)didReceiveMemoryWarning {
+  [super didReceiveMemoryWarning];
+}
+
+- (NSString*)didReceiveString:(NSString*)message {
+  [self.nativeViewController didReceiveIncrement];
+  return emptyString;
+}
+
+- (void)prepareForSegue:(UIStoryboardSegue*)segue sender:(id)sender {
+  self.messageName = @"increment";
```

Comment:
Maybe I am not understanding what you're doing here, but it seems like the message name should be const.